packages/utils/logger/src/formats/exclude-colors.ts

```diff
@@ -1,10 +1,14 @@
-import { format } from 'winston';
+import { format, Logform } from 'winston';
 
 /**
  * This will remove the chalk color codes from the message provided.
  * It's used to log plain text in the log file
  */
-export default format.printf(({ message }: { message: string }) => {
+export default format.printf(({ message }: Logform.TransformableInfo) => {
```

Comment:
You don't need to explicitly add the parameter type here as it is part of the logform printf declaration
```ts
function printf(templateFunction: (info: TransformableInfo) => string): Format;
```
```suggestion
export default format.printf(({ message }) => {
```

Reply:
Yeah indded I didn't even think of it removing the older annotation nice catch